packages/build/src/core/flags.js

```diff
@@ -1,5 +1,9 @@
 /* eslint eslint-comments/no-use: off, max-lines: off */
 
+const jsonParse = function (value) {
+  return value === undefined ? undefined : JSON.parse(value)
```

Comment:
The new version of `yargs` calls `coerce()` even when the CLI flag has not been provided.